alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/java/org/alfresco/transformer/LibreOfficeController.java

```diff
@@ -79,7 +79,7 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(LibreOfficeController.class);
 
-    @Value("${transform.core.libreoffice.home}")
```

Comment:
intentional?

Reply:
Yes, both the AIO controller and the LibreOffice controller uses an application.yml file to define this variable. As they were given different variable names it prevented inheritance of the test class.

Reply:
Ah yes they should be the same but right now in aio application.yml it is libreoffice.path and in libreoffice it is libreoffice.home. @KikoUK any one in particular it should be?

Reply:
I tried to keep it consistent as requested by Jan in these comments: https://github.com/Alfresco/alfresco-transform-core/pull/203#discussion_r408943587
https://github.com/Alfresco/alfresco-transform-core/pull/203#discussion_r408944665
I kept the original name of the var OFFICE_HOME everywhere, but then there were a few requests to change it to path. I agree that it needs to be the same in both aio and libreoffice, I guess we need to agree on which name to use. I vote for OFFICE_HOME as the method which sets it is called setOfficeHome. So ..office.home